src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -78,142 +115,131 @@ const PublicationForm = ({
-                          open={
-                            searchTerm !== '' ||
-                            theme.topics.some(topic =>
-                              topic.publications.some(
-                                publication =>
-                                  publication.id === values.publicationId,
-                              ),
-                            )
-                          }
-                        >
-                          {theme.topics.map(topic => (
-                            <DetailsMenu
-                              summary={topic.title}
-                              key={topic.id}
-                              id={`${formId}-topic-${topic.id}`}
-                              detailsId="topic"
-                              open={
-                                searchTerm !== '' ||
-                                topic.publications.some(
-                                  publication =>
-                                    publication.id === values.publicationId,
-                                )
-                              }
-                            >
-                              <FormFieldRadioGroup
-                                legend={`Choose option from ${topic.title}`}
-                                legendHidden
-                                small
-                                showError={false}
-                                name="publicationId"
-                                disabled={form.isSubmitting}
-                                options={topic.publications.map(
-                                  publication => ({
-                                    label: publication.title,
-                                    value: publication.id,
-                                  }),
-                                )}
-                              />
-                            </DetailsMenu>
-                          ))}
-                        </DetailsMenu>
-                      ))
-                    ) : (
-                      <p>No publications found</p>
+                <p>or</p>
+
+                <div className={styles.optionsContainer}>
+                  <FormRadioGroup
+                    id={`${formId}-themes`}
+                    legend="Select a theme"
+                    legendSize="s"
+                    name="themeId"
+                    small
+                    options={themes.map(theme => {
+                      return {
+                        label: theme.title,
+                        value: theme.id,
+                      };
+                    })}
+                    value={selectedThemeId}
+                    onChange={e => {
+                      setSelectedThemeId(e.target.value);
+                      form.setFieldValue('publicationId', '');
+                      setSearchTerm('');
+                    }}
+                  />
+
+                  <div className={styles.publicationsList} id="publications">
```

Comment:
This doesn't seem like a very safe id. Couldn't the skip link use the `#${formId}-publications` to jump to the radio group?